tests/smoke/README.md

```diff
@@ -0,0 +1,17 @@
+# Addons-Frontend Smoke Tests
+
+A simple smoke test suite for the addons-frontend docker image.
+
+### Prerequisites
+
+Python 3 is recommended but the tests can run on Python 2. You must have `tox` installed either way.
+
+You should also stop any services that may be using the port `3000`.
+
+### Steps
+
+1. Build the docker image from the projects root directory: `docker build -t addons-frontend .`.
+2. Run the script `setup_docker.sh` in the `tests/smoke` directory.
```

Comment:
This would be easier to copy/paste as: Run the script `./tests/smoke/setup_docker.sh`

Reply:
This should also say: ...and wait for it to exit.

I'm suggesting that because it takes a really long time.